src/shell/scripts/omp.lua

```diff
@@ -207,17 +258,47 @@ end
 
 -- Tooltips
 
+local function get_tip_command(line)
+    if USE_ENTIRE_COMMAND_LINE then
+-- REVIEW:  This is what oh-my-posh was doing -- was that intentional?
```

Comment:
I would say it makes sense to only used the second one as we only need the first keyword. So yes, we can remove this in theory.

Reply:
@chrisant996 👀  the first one can be removed if the second one works.